documentation/abnormal_termination/README.md

```diff
@@ -39,11 +43,49 @@ To find out if the process exited because of a `process.exit()` call, follow [Us
 
 ### Process exits with Native Stack Trace
 
-TBD
+Some Node.js application uses native modules to re-use existing components or
+performance optimize specific areas. A common example for native modules are
+database drivers. In the case our native module has an issue it can crash the
+Node.js application which will generate a native stack trace.
 
-### Process exits with unmeaningful Stack Trace
+You can find the most common scenarios for native crashes here:
 
-TBD
+1. Native module crashes due bug
```

Comment:
This is quite generic and includes point 2 and 3.

Reply:
How about if the points would be:

1. Operational error in Node.js runtime and native dependencies
  - For example, V8 checks, when input checking is missing can lead to a crash
2. Programmer error in native code
  - For example, accessing incorrect memory address
3. Native module crashes due to memory leak (native memory leak)

Reply:
Yes thats better. Other cases I have seen several times:
* Access garbage collected data
* Call v8 APIs without `HandleScope`
* Call v8 APIs from wrong Thread

Reply:
Thank you, added